plugins/maciVoting/services/coordinator/utils/schemas.ts

```diff
@@ -0,0 +1,74 @@
+import { z } from "zod";
```

Comment:
I have never used zod but it looks useful 😊 . Let's keep it

Reply:
I've just done some more reading and I'm starting to think it might actually be over-engineering for this use-case 😂

You [should use zod](https://www.totaltypescript.com/when-should-you-use-zod) when you do not trust the data coming into your app. In this case, we do trust the data coming into our app as we control the API it is calling.